### PR TITLE
Express route stats cmdlets

### PR DESCRIPTION
**Pull request by dihan0604**

### CODE REVIEW

**Comment by hovsepm** [comment]
on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/859/

**Comment by dihan0604** [comment]
On-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/862/

**Comment by dihan0604** [comment]
On demand job passed: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/866/

File: src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/GetAzureExpressRouteARPTableCommand.cs [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Commands.Network.Models;
+using MNM = Microsoft.Azure.Management.Network.Models;
+
+namespace Microsoft.Azure.Commands.Network
+{
+     [Cmdlet(VerbsCommon.Get, "AzureRmExpressRouteCircuit"), OutputType(typeof(PSExpressRouteCircuit))]
+    public class GetAzureExpressRouteCircuitCommand : ExpressRouteCircuitBaseCmdlet
```
Comment (hovsepm) [inline src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/GetAzureExpressRouteARPTableCommand.cs]: please fix indentation.

File: src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Stats/GetAzureExpressRouteCircuitARPTableCommand.cs [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+ using AutoMapper;		
+ using Microsoft.Azure.Commands.Tags.Model;		
+ using Microsoft.Azure.Management.Network;		
+ using Microsoft.Azure.Commands.Network.Models;		
+ using MNM = Microsoft.Azure.Management.Network.Models;		
+ 		
+ namespace Microsoft.Azure.Commands.Network		
+ {		
+     using System;		
+     using System.Collections;
```
Comment (hovsepm) [inline src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Stats/GetAzureExpressRouteCircuitARPTableCommand.cs]: move these usings above namespace declaration.

File: src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Stats/GetAzureExpressRouteCircuitRouteTableCommand.cs [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+ using AutoMapper;		
+ using Microsoft.Azure.Commands.Tags.Model;		
+ using Microsoft.Azure.Management.Network;		
+ using Microsoft.Azure.Commands.Network.Models;		
+ using MNM = Microsoft.Azure.Management.Network.Models;		
+ 		
+ namespace Microsoft.Azure.Commands.Network		
+ {		
+     using System;		
+     using System.Collections;
```
Comment (hovsepm) [inline src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Stats/GetAzureExpressRouteCircuitRouteTableCommand.cs]: move these usings above namespace declaration.

File: src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Stats/GetAzureExpressRouteCircuitRouteTableSummaryCommand.cs [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+using AutoMapper;
+using Microsoft.Azure.Commands.Tags.Model;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Commands.Network.Models;
+using MNM = Microsoft.Azure.Management.Network.Models;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    using System;
+    using System.Collections;
```
Comment (hovsepm) [inline src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Stats/GetAzureExpressRouteCircuitRouteTableSummaryCommand.cs]: move these usings above namespace declaration.

File: src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Stats/GetAzureExpressRouteCircuitRouteTableSummaryCommand.cs [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+        [ValidateNotNullOrEmpty]
+        public string DevicePath { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            base.ExecuteCmdlet();
+            DevicePathEnum path;
+            if (Enum.TryParse(DevicePath, true, out path))
+            {
+                var arpTables = this.NetworkClient.NetworkManagementClient.ExpressRouteCircuits.ListRoutesTableSummary(ResourceGroupName, ExpressRouteCircuitName, PeeringType, DevicePath).Value.Cast<object>().ToList();
```
Comment (hovsepm) [inline src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Stats/GetAzureExpressRouteCircuitRouteTableSummaryCommand.cs]: this line is too long. please restructure the code in a way that it does not exceed ~120 symbols per line.

File: src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Stats/GetAzureExpressRouteCircuitARPTableCommand.cs [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+         [ValidateNotNullOrEmpty]		
+         public string DevicePath { get; set; }		
+ 		
+         public override void ExecuteCmdlet()		
+         {		
+             base.ExecuteCmdlet();		
+             DevicePathEnum path;		
+             if (Enum.TryParse(DevicePath, true, out path))		
+             {		
+                 var arpTables = this.NetworkClient.NetworkManagementClient.ExpressRouteCircuits.ListArpTable(ResourceGroupName, ExpressRouteCircuitName, PeeringType, DevicePath).Value.Cast<object>().ToList();
```
Comment (hovsepm) [inline src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Stats/GetAzureExpressRouteCircuitARPTableCommand.cs]: this one as well - line is too long to view on github.

**Comment by dihan0604** [comment]
I have addressed the comments, please merge the PR, thanks !

**Pull request merged**